xml/chapter5/section5/section5.xml

```diff
@@ -455,10 +606,10 @@ assign("fun", list(op("lookup_symbol_value"), constant("f"), reg("env")))
     </SPLITINLINE>
     search.  We will discuss how to implement such
     <SPLITINLINE>
-      <SCHEME>variable</SCHEME>
-      <JAVASCRIPT>name</JAVASCRIPT>
```

Comment:
In the case of lexical addressing, the term "name access" is misleading, because there are no "names" or "symbols" at run time.